03-make/make_soup/Makefile

```diff
@@ -0,0 +1,43 @@
+
+VEGETABLES=kitchen/fridge/carrot\
+ kitchen/fridge/potato\
+ kitchen/fridge/onion
+
+MEAT=kitchen/fridge/meat
+
+KNIFE=kitchen/table/knife
+GAS_COOKER=kitchen/gas_cooker
+POT=kitchen/table/pot
+POT_SOUP=dining-room/pot
+
+CHOP=./chop.sh
+BUY=./move_resource.sh
+
+cook_soup: $(POT) add_water chop_vegetables chop_meat $(GAS_COOKER)
+	@echo "Now it's tasty soup" >> $(POT) &&\
+	mv $(POT) $(POT_SOUP)
+
+#implicit rules for shopping 
+kitchen/fridge/% : food-market/%
+	@touch $@ && \
```

Comment:
You can use <tab> in the beginning of line instead of line concatenation with symbol \